src/components/Footer/Footer.scss

```
@@ -0,0 +1,79 @@
+footer hr {
+  margin: 20px 50px 50px 50px;
+}
```

```suggestion
footer {
  hr {
  margin: 20px 50px 50px 50px;
}
```
footer 선택자로 전체 스타일 속성을 감싸는 nesting이 필요해 보입니다.
scss로 스타일을 구현할 때는 html 구조와 비슷하게 구성을 짜는게
추후 유지보수에 더 도움이 됩니다!

---

네 멘토님 해당 내용 확인하여 다시 수정하였습니다. 확인 부탁드립니다. 감사합니다!